packages/web/components/swap-tool/alt.tsx

```diff
@@ -94,8 +94,18 @@ export const AltSwapTool: FunctionComponent<SwapToolProps> = observer(
     // auto focus from amount on token switch
     const fromAmountInputEl = useRef<HTMLInputElement | null>(null);
 
+    const outputDifference = new RatePretty(
+      swapState.inAmountInput?.fiatValue
+        ?.toDec()
+        .sub(swapState.tokenOutFiatValue?.toDec())
+        .quo(swapState.inAmountInput?.fiatValue?.toDec()) ?? new Dec(0)
+    );
+
+    const showOutputDifferenceWarning =
+      outputDifference.toDec().abs().gt(new Dec(0.05)) ?? false;
```

Comment:
nit: is this coalescing operator necessary?

Reply:
```suggestion
      outputDifference.toDec().abs().gt(new Dec(0.05));
```